modin/_compat/pandas_api/abc/dataframe.py

```diff
@@ -0,0 +1,45 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module for interface for class compatibility layer for DataFrame."""
+
+from modin.logging import ClassLogger
+from .base import BaseCompatibleBasePandasDataset
+
+
+class BaseCompatibleDataFrame(BaseCompatibleBasePandasDataset, ClassLogger):
```

Comment:
Do we need to inherit the ClassLogger here if its already inherited by the `BaseCompatibleBasePandasDataset`?

Reply:
we don't, I guess this is a leftover from dataset not inheriting from ClassLogger...